components/help.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useContext } from "react";
+
+import { LanguageContext } from "@contexts/translator";
+
+// eslint-disable-next-line react/prop-types
+const Help = ({ close }) => {
+  const { getTranslation } = useContext(LanguageContext);
+
+  return (
+    <div id="help-info" className="help-info padded spaced-lines">
+      <button
+        id="help-info-close-button"
+        className="help-info-close-button"
+        aria-label="Close help info section"
+        onClick={close}
+      >
+        X{" "}
+        <span
+        // data-translation-id="close"
+        >
+          {getTranslation("close")}
+        </span>
+      </button>
+      <p className="p txt-small">
+        <span
+          // data-translation-id="project_description"
+          // I don't love this but alas
+          dangerouslySetInnerHTML={{
+            __html: getTranslation("project_description"),
+          }}
+        />
+        .
+      </p>
+      <br />
+      <p className="p txt-small bold">
+        <span
+        // data-translation-id="project_feedback"
+        >
+          {getTranslation("project_feedback")}
+        </span>{" "}
+        <a
+          href="mailto:support@tcmap.org"
+          // data-translation-id="project_contact"
+        >
+          {getTranslation("project_contact")}
+        </a>
+        .
+      </p>
+      <p className="p txt-small bold">
+        <span
+        // data-translation-id="project_data"
+        >
+          {getTranslation("project_data")}
+        </span>{" "}
+        <a
+          href="https://airtable.com/shr2el3WSJHLNgQUx/tblGDXjVZuA2GejcN"
+          target="_blank"
+          // onClick="captureOutboundLink('https://airtable.com/shr2el3WSJHLNgQUx/tblGDXjVZuA2GejcN', 'airtable')"
```

Comment:
Does this no longer track clicks to the link then?

Reply:
Yeah I haven't reimplemented the Google Analytics tracking at all, but I'm sure we could add some onClick handler function that we pass down to here! Per our discussion I think I'd like to rip out GA entirely and go with [Netlify Analytics](https://www.netlify.com/products/analytics/) instead, but happy to discuss that further!

Reply:
It all depends on how much we currently need GA! If we're not super tied down to it, I'd rather just ship this clean without GA. Netlify Analytics won't even require any code changes, just setting up some stuff in the Netlify dashboard!

Reply:
I'm hesitant to just gut it because then there will be a gap in metrics... Esp depending on how long it might take to set up NA... also though.. I was looking and.. is Netlify Analytics only a paid plan?